doc/status.md

```diff
@@ -2,3 +2,135 @@
 Status
 ======
 
+
+Disk
+----
+
+Implemented:
+
+* Probing.
+
+Missing:
+
+* BIOS-ID handling.
+
+
+DASD
+----
+
+Missing.
+
+
+Partition Table
+---------------
+
+Implemented:
+
+* For MS-DOS and GPT creating and deleting, resizing partitions.
```

Comment:
NP, just the `and` position and add parenthesis like in:

https://github.com/openSUSE/libstorage-ng/pull/156/files#diff-55cc77d885cd4191477652d3b2b3360dR115

For MS-DOS and GPT (creating, deleting and resizing partitions).